Assets/Samples/ProjectWideActionsSample/ProjectWideActionsSample.cs

```diff
@@ -0,0 +1,46 @@
+#if UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
+
+namespace UnityEngine.InputSystem.Samples.ProjectWideActionsSample
+{
+    public class ProjectWideActionsSample : MonoBehaviour
+    {
+        [SerializeField] public GameObject cube;
+
+        // Start is called before the first frame update
+        void Start()
+        {
+        }
+
+        // Update is called once per frame
+        void Update()
+        {
+            // Project-Wide Actions via Source Generated type-safe API
+            if (ProjectActions.player.attack.wasPressedThisFrame)
+            {
+                cube.GetComponent<Renderer>().material.color = Color.red;
+            }
+            else if (ProjectActions.player.attack.wasReleasedThisFrame)
+            {
+                cube.GetComponent<Renderer>().material.color = Color.green;
+            }
+
+            if (ProjectActions.player.move.value.x < 0.0f)
```

Comment:
Not high priority to address on this branch and not really introduced here, but as I commented on a previous PR this can be replaced by a one or two liner to simplify sample:

`cube.transform.Translate(ProjectActions.player.move.value * Time.deltaTime);` 

Maybe we should also add a comment after the translate as such:
// Multiplying with Time.deltaTime since assuming action binding values are absolute and not relative

Reply:
(Such a change could have been outside TypeSafe API branch and added to develop since not relating to that functionality?)